### DISCUSSION TITLE
External German Keyboard - Pipe symbol non functional

### DISCUSSION THREAD

**Discussion by chrgraf** [blinksh/blink#1916] [Support]
Dear all,

I am using Blink Shell v17.1.0.854 with an external Bluettoth Keyboard.
Outside blink, the Pipe Symbol works fine with pressing option/alt + 7.

But when launching Blink Shell, option + 7 does not any longer work and I am wondering what I miss in the config to get it working.

![IMG_2267](https://github.com/blinksh/blink/assets/22005482/15eae5b6-faa0-475c-8a1f-c20bd0e148dd)

I tried as well a Custom Press in this way:
![IMG_2268](https://github.com/blinksh/blink/assets/22005482/82dccf1a-f683-407d-b51d-d2cba532fea6)

Interesting wise, the Showkey does not trigger when pressing the option key/alt key. Tried some other keys to see if working in general, e.g. option + n to get the tilde ~ is doing fine.

blink> showkey
Press any keys - Ctrl-D will terminate this program.
1        49 0061 0x31
2        50 0062 0x32
3        51 0063 0x33
\t        9 0011 0x09
^[       27 0033 0x1b
<        60 0074 0x3c
y       121 0171 0x79
~       126 0176 0x7e.            < Tilde with option +7 works fine
^M       13 0015 0x0d
~       126 0176 0x7e
^M       13 0015 0x0d Platform is IPAD Pro, 4th generation

Any further help would be appreciated.

Thanks

**Comment by carloscabanero** [comment]
Hi! Thanks for reaching out. Can you send me the output of https://jsfiddle.net/carloscabanero/q8rs1ftg/? Thanks!

**Comment by chrgraf** [comment]
Dear Carlos,

Thanks. Please find the output below, both as screenshot where it shows the pipe and the markdown.


typed text:

    |

| n |        type |     code | key |  kc | mods | repeat | l | comp |      iType | data |
|--:|------------:|---------:|:----|----:|-----:|-------:|--:|-----:|-----------:|-----:|
| 1 |     keydown | AltRight | Alt |  18 |    A |        | 2 |      |            |      |
| 2 |     keydown |   Digit7 | |   | 220 |    A |        | 0 |      |            |      |
| 3 | beforeinput |          |     |     |      |        |   |      | insertText |    | |
| 4 |       input |          |     |     |      |        |   |      | insertText |    | |
| 5 |       keyup |   Digit7 | |   | 220 |    A |        | 0 |      |            |      |
| 6 |       keyup | AltRight | Alt |  18 |      |        | 2 |      |            |      |
| 7 |      change |          |     |     |      |        |   |      |            |      |


<img width="398" alt="image" src="https://github.com/blinksh/blink/assets/22005482/4ca82e15-1935-436f-9919-71eaf64a998f">

**Comment by mutamamQ** [comment]
Hello I am having the same problem on a turkish magic keyboard.
